recipes/omero-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "omero-py" %}
+{% set version = "5.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omero-py-{{ version }}.tar.gz
+  sha256: 4fbf7288586093d45e8f3a661ddf46143f38231017fada39c4e1c26725b34e33
+
+build:
+  entry_points:
+    - omero=omero.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - appdirs
+    - future
+    - numpy
+    - pillow
+    - python
+    - pywin32  # [win]
+    - pyyaml
+    - requests
+    - zeroc-ice <3.7
+
+test:
+  imports:
```

Comment:
Perhaps add the full list of imports from
https://github.com/conda-forge/staged-recipes/blob/569b10b8b6fe67c1d2be13ad18c75892bff2b381/recipes/omero-py/meta.yaml#L46-L59
though you'll need to check whether that list is up to date.